docs/pages/kubernetes-access/guides.mdx

```diff
@@ -8,3 +8,4 @@ layout: tocless-doc
 - [Teleport Kubernetes Access and Trusted Clusters](./guides/federation.mdx): Federated Access using Teleport Trusted Clusters.
 - [Kubernetes Access Multiple Clusters](./guides/multiple-clusters.mdx): Connecting Multiple Clusters with Teleport Kubernetes Access.
 - [Kubernetes Access from standalone Teleport](./guides/standalone-teleport.mdx): Connecting standalone Teleport installations to Kubernetes clusters
+- [Kubernetes Auto-discovery](./guides/discovery.mdx): How to configure Teleport to automatically discover Kubernetes clusters
```

Comment:
What would you think of adding this to its own subsection of "Kubernetes Access", rather than the "Guides" section? This way, we don't need to introduce a new subsection depth level.